brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-2021 Fix: WP 6.1 font size and font weight CSS conflicting spectra option labels.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-2021
- Core category for Info Box and Call To Action.
- Fix wordpress conflicting with label font-size and text-transform. 

### Screenshots

- CSS conflict fix - https://share.getcloudapp.com/p9uQJ0Ob
- Admin categories - https://share.getcloudapp.com/o0uXdW7N

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop the block and check label for control like info box - Call To Action - Link label.
- Go to the Admin page, select core categories, and verify info-box and CTA is visible in core categories.

### Checklist:
